server/server/common/queries.py

```diff
@@ -1001,3 +1001,12 @@ class PWA:
     LIMIT 1
     RETURN {@uid: doc.filepath}
 '''
+
+SUTTA_PALI_REFERENCE = '''
+FOR pali in pali_reference_edition
+    Collect edition_set = pali.edition_set, name = pali.name, short_name = pali.short_name
```

Comment:
and one more thing, my mistake, I missed it last time, sorry =(
but I think we should write keywords in AQL in caps, because we do that everywhere (`in` => `IN`, `Collect` => `COLLECT`, etc.)

Reply:
Fixed in e7658b8